web-app/src/translations/en.tsx

```diff
@@ -552,10 +552,14 @@ export default {
 		},
 		overview_tab: {
 			apply_now: "Apply Now",
+<<<<<<< HEAD:web-app/src/translations/en.tsx
```

Comment:
You have a merge conflict that did not get fixed.

Reply:
Just fixed it!